ingestion-beam/src/main/java/com/mozilla/telemetry/util/KeyStore.java

```diff
@@ -0,0 +1,122 @@
+package com.mozilla.telemetry.util;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.google.api.pathtemplate.ValidationException;
```

Comment:
I think this is importing the wrong class.

Reply:
Interesting that this doesn't fail, since I do check for the ValidationException in the unit tests.

Reply:
This is because everit's ValidationException extends IOException, so we successfully catch it anyway without explicitly listing it in the signature.